packages/wallet/src/redux/protocols/existing-ledger-funding/reducer.ts

```diff
@@ -105,10 +77,107 @@ export const existingLedgerFundingReducer = (
       return waitForPostFundSetupReducer(protocolState, sharedData, action);
     case 'ExistingLedgerFunding.WaitForLedgerTopUp':
       return waitForLedgerTopUpReducer(protocolState, sharedData, action);
+    case 'ExistingLedgerFunding.WaitForChannelSync':
+      return waitForChannelSyncReducer(protocolState, sharedData, action);
+    case 'ExistingLedgerFunding.WaitForChannelLock':
+      return waitForChannelLockReducer(protocolState, sharedData, action);
```

Comment:
With channel locking protocols now need to wait until they can acquire a lock on a channel before attempting to work with it. Currently this is only implemented in the `existingChannelReducer`